src/legacy/core_plugins/vis_default_editor/public/schemas.ts

```diff
@@ -46,8 +45,7 @@ export interface Schema {
 }
 
 export class Schemas {
-  // @ts-ignore
-  all: IndexedArray<Schema>;
+  all: Schema[] = [];
```

Comment:
👍 Done.
Since now `Schemas` has the same properties as `ISchemas` has, I got rid of `ISchemas`.

Reply:
hmm, AFAIK `ISchemas` was intentionally created to use an interface (there were a lot of places with such an approach, so all of interfaces starts with `I` ). 
@flash1293 am I right?

Reply:
@stacey-gammon When I remember correctly you started to remove the types separate to actual class implementations in some places, right? Do you know what the current best practices around this are?

Reply:
For awhile we (app arch) were moving toward an `I` prefix convention in these scenarios, but have since stopped doing this in favor of waiting for type-only imports [which are available in TS 3.8](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html#type-only-imports-exports). I believe this is the approach the platform team is taking too.

In the meantime, for the cases where we need a separate type in the interim, we have been inconsistent, using `I*` in some places but not in others.

Reply:
In that case I like the current solution, once TS 3.8 is merged, we can do a bulk change of the remaining places.

Reply:
For those who want to follow along, here's the PR which is working toward upgrading TS in Kibana: https://github.com/elastic/kibana/pull/57774

Reply:
Discussion here if you are interested on the I-prefix: https://github.com/elastic/kibana/issues/51674